flink-python/pyflink/table/table_environment.py

```diff
@@ -800,14 +806,20 @@ def _from_file(self, filename, schema):
         gateway = get_gateway()
         blink_t_env_class = get_java_class(
             gateway.jvm.org.apache.flink.table.api.internal.TableEnvironmentImpl)
-        if blink_t_env_class == self._j_tenv.getClass():
-            raise NotImplementedError("The operation 'from_elements' in batch mode is currently "
-                                      "not supported when using blink planner.")
+        is_blink = (blink_t_env_class == self._j_tenv.getClass())
+        row_type_info = _to_java_type(schema)
+        if is_blink:
+            execution_config = gateway.jvm.org.apache.flink.api.common.ExecutionConfig()
```

Comment:
Could you add some comments describing why the blink planner should be processed differently?